tests/tasks/test_celery.py

```diff
@@ -1,10 +1,10 @@
-def test_create_task(celery_app, celery_worker):
-    @celery_app.task
+def test_create_task(celery_session_app, celery_session_worker):
```

Comment:
`celery_app` and `celery_worker` are function scoped fixtures, so a worker is started on a new thread for each function, which is very slow. `celery_session_app` and `celery_session_worker` are session scoped fixtures, so we'll only use one worker per test session, which is much faster.